examples/skr/aks/README.md

```diff
@@ -1,4 +1,8 @@
-# Microsoft Secure Key Release (SKR) AKS Example
+# NOTE This example is for te Kata based AKS preview
+#      Today the supported GA solution for Kubernetes is to use virtual nodes.
+
+
+# Microsoft Secure Key Release (SKR) AKS Confidential Pods (ie based on Kata) Example
```

Comment:
*i.e.

Reply:
Sorry? Can you elaborate please?

Reply:
Just a nit about the correct punctuation: i.e. instead of ie in the parenthesis.